canonical-web-and-design/snapcraft.io
# Move usabilla code into Raven context to avoid script blocking

## Done
Moved the usabilla code snippet into a Raven context to prevent the script being blocked

## QA
- Go to https://snapcraft-io-3706.demos.haus/admin/ahnuP3quahti9vis8aiw/members/invites
- Check in the network tab that the usabilla script is not blocked
- Compare to live: https://snapcraft.io/admin/ahnuP3quahti9vis8aiw/members/invites
